composer.json

```diff
@@ -20,16 +20,10 @@
       "email": "luke@fourkitchens.com"
     }
   ],
-  "repositories": [
-    {
-      "type": "composer",
-      "url": "https://packages.drupal.org/8"
-    }
-  ],
+  "repositories": [],
   "require": {},
   "require-dev": {
-    "drupal/coder": "2.7",
-    "squizlabs/PHP_CodeSniffer": "~2.0"
```

Comment:
@generalredneck I get the following error when running `gulp phpcs` without specifying version 2:
```
Fatal error: Interface 'PHP_CodeSniffer_Sniff' not found in /path/to/node_modules/project-gulp/vendor/drupal/coder/coder_sniffer/Drupal/Sniffs/Array/ArraySniff.php on line 24
```